lib/api/calendars.ts

```diff
@@ -51,3 +51,12 @@ export const modifyEvent = (client: DAVClient) => async (eventPath: string, even
     body: translate(event)
   });
 }
+
+export const changeParticipation = (client: DAVClient) => async (eventPath: string , mailAttendee:string,action:string , event:CalendarEventObject): Promise<string> => {
+  if(event.attendees){
+    const  attendeeIndex = event.attendees.findIndex((attendee => attendee.email == mailAttendee));
+    event.attendees[attendeeIndex].partstat = action;
+  }
+
+  return  await modifyEvent(client)(eventPath,event);
```

Comment:
```suggestion
  
  return  await modifyEvent(client)(eventPath,event);
  }
```

no point in calling modifyEvent if nothing changed